src/App.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+import { Route, Routes } from 'react-router-dom';
+import Home from './components/Home';
+import './App.css';
+import Layout from './components/Layout';
+import Signup from './components/Signup';
+import Login from './components/Login';
+import Details from './components/Details';
+import Reserve from './routes/Reserve';
+import MyReservations from './components/MyReservations';
+import AddMotorbikeForm from './components/AddMotorbikeForm';
+import DeleteMotorbike from './components/DeleteMotorbike';
+
+function App() {
```

Comment:
- [ ] Your API is to be consumed on port 4000:

![image](https://github.com/SaneMyburg/motorbike-front-end/assets/57421823/bed16dd8-8e33-423b-aefa-aab39d4ac5ba)

Kindly, make your instructions on the backend reflect this so your users can run your project smoothly. 👍 

```
rails -p 4000
```